drainer/translator/mysql.go

```diff
@@ -264,7 +273,20 @@ func formatData(data types.Datum, ft types.FieldType) (types.Datum, error) {
 			return types.Datum{}, err
 		}
 		data = types.NewUintDatum(val)
+	case mysql.TypeTinyBlob, mysql.TypeMediumBlob, mysql.TypeLongBlob, mysql.TypeBlob:
+		//only for oracle db
+		if destDBType == loader.OracleDB && isBlob(ft) {
+			data = types.NewBytesDatum(data.GetBytes())
+		}
 	}
 
 	return data, nil
 }
+
+func isBlob(ft types.FieldType) bool {
+	stype := types.TypeToStr(ft.Tp, ft.Charset)
+	if stype == "blob" || stype == "tinyblob" || stype == "mediumblob" || stype == "longblob" {
```

Comment:
```suggestion
	if ft.Tp == mysql.TypeBlob || ft.Tp == mysql.TypeTinyBlob || ft.Tp == mysql. TypeMediumBlob || ft.Tp == mysql.TypeLongBlob {
```

Reply:
if do that, it cannot distinguish blob and text